vendor.conf

```diff
@@ -78,7 +78,7 @@ google.golang.org/grpc v1.12.0
 # the containerd project first, and update both after that is merged.
 # This commit does not need to match RUNC_COMMIT as it is used for helper
 # packages but should be newer or equal.
-github.com/opencontainers/runc 96ec2177ae841256168fcf76954f7177af9446eb
+github.com/opencontainers/runc 12f6a991201fdb8f82579582d5e00e28fba06d0a
```

Comment:
note that this is only the vendored code; this version matches the version that's used in containerd 1.2.3. The runc _binary_ is updated in a separate PR to address the CVE